servers/dsub/jobs/controllers/aggregation_controller.py

```diff
@@ -24,12 +25,79 @@ def get_job_aggregations(timeFrame, projectId=None):
     Returns:
         AggregationResponse: Response contains aggregation of jobs
     """
+    query = QueryJobsRequest.from_dict({
+        'extensions': {
+            'project_id': projectId
+        }
+    })
+
+    query.start = time_convert.time_frame_to_start_time(timeFrame)
```

Comment:
We want the summaries to include all jobs that are still running or ended within the requested time period. This is not the same as filtering on the query start time.

For example, if I start a job 2 days ago and it's still running, then use TIME_FRAME = 1D, I should see the job in the summary.